src/utils/index.js

```diff
@@ -215,3 +215,20 @@ export const getValuesToUpdate = (balances, baseBalances, update) => {
   });
   return result;
 };
+
+
+export const removePrefix = (string, prefix = 'OPEN.') => {
```

Comment:
These functions should not be done here, they are the part of the UI needed only for display purposes. We should keed the data raw in vuex-bitshares. The same applies to date-fns.